### ISSUE TITLE
undo glitch 

### ISSUE THREAD

**Issue by Dixxi0509** [tahoma2d/tahoma2d#791]
the undo button keeps glitching out you see when i use tahoma2d [i have the 1.1 version because my computer won't let me get the 1.2] the undo button would glitch i was animating one time and when i hit undo it messes up my frames below when im above them i don't know how to fix it

**Comment by DarrenTAnims** [comment]
Yes, it does that. If you have repeatable steps, then please do share them . Otherwise, just save your project and restart Tahoma to fix it.

**Comment by Dixxi0509** [comment]
Ok thanks but a few things I don't know when it's gonna happen and when it
does happen i would just get rid of the projecy due t it messing up my
frames

**Comment by artisteacher** [comment]
Yeah, the fact that we currently don't know when undo history will get messed up makes it very challenging to track and fix. If anyone ever reports specific steps, we're much more likely to get a fix.

**Comment by Dixxi0509** [comment]
Do they plan on fixing it I love the soft but due to this it's challenging
I would download the older versions thinking that I broke it when it first
happened I thought I messed something up

**Comment by DarrenTAnims** [comment]
Unless someone can find a repeatable problem, it's not fixable. And I don't think any devs are investigating the undo problems atm.

**Comment by Dixxi0509** [comment]
Ok I get that but it's just annoying how it does that (the undo button)

**Comment by DarrenTAnims** [comment]
Agreed. But some parts of Tahoma we've just got to accept that that's the way they are. And the undo system is one of those.

**Comment by Dixxi0509** [comment]
This wasn't in the contract (XD)

**Comment by manongjohn** [comment]
Just to be clear...

It's not that I don't want to fix it.  I very much want to fix it, but it's hard to fix something I cannot easily replicate.  

More than likely there are multiple reasons under certain scenarios that could cause Undo to break which is probably why it is difficult to pin down.

**Comment by Dixxi0509** [comment]
Hey it's ok I understand to be honest I'll just leave it be while it is
annoying I just have t deal with what I got

**Comment by gleek12** [comment]
Hello, Im new to the app, but I've been seeing this bug as well.  I'm having trouble pinning it down, but it seems to be related (for me) to using the eraser tool on raster levels.  If I dont use the dedicated eraser tool, but instead use the eraser brushes I havent yet seen the undo bug manifest.  Don't know if that helps, I will report more if I can narrow it down further.
